engines/dfc_provider/app/services/order_builder.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class OrderBuilder < DfcBuilder
+  def self.build_anonymous(order)
+    id = urls.anonymous_orders_url
+
+    DataFoodConsortium::Connector::Order.new(
+      id,
+      orderStatus: 'complete'
+    ).tap do |e|
+      add_ofn_property(e, "ofn:producer_postcode", order.producer_postcode)
+      add_ofn_property(e, "ofn:distributor_postcode", order.distributor_postcode)
+      add_ofn_property(e, "ofn:variant_unit_name", order.unit_name)
+      add_ofn_property(e, "ofn:variant_unit_type", order.unit_type)
+      add_ofn_property(e, "ofn:variant_units", order.units)
+      add_ofn_property(e, "ofn:price", order.price.to_f)
+      add_ofn_property(e, "ofn:quantity_sold", order.quantity_sold)
```

Comment:
I think that for this project, we want to be compatible with another software exporting DFC format as well. They don't have any OFN properties. So we can't use them either and need to express everything in the DFC format. For example, the postcodes can be included in address objects and they can be added to the export by passing the address objects as paramters to the export method.